AdonisUI.ClassicTheme/NamedStyles/WindowToggleButton.xaml

```diff
@@ -0,0 +1,123 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:adonisExtensions="clr-namespace:AdonisUI.Extensions;assembly=AdonisUI"
+                    xmlns:adonisUi="clr-namespace:AdonisUI;assembly=AdonisUI">
+
+    <ResourceDictionary.MergedDictionaries>
+        <ResourceDictionary Source="pack://application:,,,/AdonisUI.ClassicTheme;component/DefaultStyles/ToggleButton.xaml" />
+    </ResourceDictionary.MergedDictionaries>
```

Comment:
This can be removed, doesn't it?

Reply:
Probably, i found the same [here](https://github.com/benruehl/adonis-ui/blob/master/AdonisUI.ClassicTheme/NamedStyles/ToolbarToggleButton.xaml#L7), so i thought it might be required. I'll verify whether this is actually needed.

Reply:
It is required over there because of the `BasedOn` expression. To make sure the style is correctly based on the default style, the default style is explicitly included again. Because your style is not based on the default style, it shouldn't be necessary to include it. Except you found another reason of course.